frontend/src/pages/EditStudent.vue

```diff
@@ -109,82 +111,68 @@ export default {
           },
           studentData: {},
           languages: [],
```

Comment:
```suggestion
          languages: [],
          myUpdateStudentSuccess: false,
```
Introduce a local variable for the success/fail notification to switch on

Reply:
Do we need a local variable when we already have one in the Vuex store (`updateStudentSuccess`)? We can always refer to that directly if we need to know whether the update succeeded or failed.

Reply:
We had some difficulty getting the page to watch the Vuex variable directly. Open to better implementation suggestions.

Reply:
There might be a collision - in the PR there is both `mapState(["updateStudentSuccess"])` to get it from Vuex, and also a watched property `updateStudentSuccess`. Both of these are trying to occupy `this.updateStudentSuccess` on the component.

Reply:
we'll remove this local variable.